src/components/forecast-list.js

```diff
@@ -0,0 +1,69 @@
+import { Sparklines, SparklinesLine, SparklinesReferenceLine } from 'react-sparklines';
+import { useSelector } from 'react-redux';
+import _ from 'lodash';
+
+
+const ForecastList = () => {
```

Comment:
I see a lot of inconsistency in how you are writing functions (expression/declaration). It is good to stay consistent.